src/main/scala/org/clulab/wm/eidos/apps/ExtractJsonLDFromFile.scala

```diff
@@ -20,7 +20,7 @@ object ExtractJsonLDFromFile extends App {
   val pw = FileUtils.printWriterFromFile(outputFile)
   // 3. Extract causal mentions from the text
   val annotatedDocument = Timer.time("Just extraction") {
-    reader.extractFromText(text)
+    reader.extractFromText(text, documentCreationTime = Some("2018-09-07"))
```

Comment:
What is this arbitrary date?

Reply:
Thanks.  Leftover from testing.  Removing...